coalib/misc/Exceptions.py

```diff
@@ -17,7 +18,8 @@ def get_exitcode(exception, log_printer=None):
     elif isinstance(exception, SystemExit):
         exitcode = exception.code
     elif isinstance(exception, VersionConflict):
-        log_message = Constants.VERSION_CONFLICT_MESSAGE % str(exception.req)
+        log_message = Constants.VERSION_CONFLICT_MESSAGE % (
+            str(exception.req), str(Bear.name()))
```

Comment:
`Bear.name()` isnt correct.  you need to put data into the `exception` instance, and then get it out here.